inst/templates/common/header.mustache

```diff
@@ -22,7 +22,7 @@
 {{#reportMetadata.comparisonSeries}}<strong>Comparison Time Series:</strong> {{reportMetadata.comparisonSeries}}{{/reportMetadata.comparisonSeries}}
 </div>
 <div class="header-col-right">
-<p><strong>Period:</strong> {{reportMetadata.startDate}} to {{reportMetadata.endDate}}</p>
+<p><strong>Period:</strong> {{reportMetadata.displayStartDate}} to {{reportMetadata.displayEndDate}}</p>
```

Comment:
I'm pretty sure there's a ticket to fix the period on the DC report so I think it's safe to assume that the period is mandatory for every report so I'm not sure if a check is really necessary. If the period is not present it's a data error so it should be okay for it to render a bit strangely to make it obvious that there is an error.